genesis/interface.go

```diff
@@ -8,6 +8,7 @@ import (
 type GenesisParser interface {
 	InitialAccountsSplitOnAddressesShards(shardCoordinator sharding.Coordinator) (map[uint32][]*InitialAccount, error)
 	InitialAccountsSplitOnDelegationAddressesShards(shardCoordinator sharding.Coordinator) (map[uint32][]*InitialAccount, error)
+	InitialAccounts() []*InitialAccount
```

Comment:
I think it would be better to return an interface instead of *InitialAccount, for all the interface methods

Reply:
The 2 structs are used as DTOs. Will add the un-necessarily interface wrapping

Reply:
done